packages/apps/halo-app/src/vault/local-vault.ts

```diff
@@ -0,0 +1,8 @@
+//
+// Copyright 2023 DXOS.org
+//
+
+import { Config, fromIFrame } from '@dxos/client';
+
+export const fromLocal = () =>
+  fromIFrame(new Config({ runtime: { client: { remoteSource: '/node_modules/@dxos/halo-app/local-vault.html' } } }));
```

Comment:
try experimenting with

```suggestion
  fromIFrame(new Config({ runtime: { client: { remoteSource: new URL('../../local-vault.html', import.meta.url).href } } }));
```

Reply:
vite processes those in a special way

Reply:
not sure I understand how those would end up pointing tho the same location